content/workload-reduction-toolkit/identify-workload-issues/staff-workload-survey.md

```diff
@@ -0,0 +1,133 @@
+                Google Forms - staff workload survey template
+              </a>
+            </li>
+          </ul>
+        </p>
+        <p>
+          Some schools find online surveys easier to share with staff than
+          a paper form because they collate the results automatically for
+          you.
+        </p>
+        <h2 class="govuk-heading-m">
+          Download the staff workload survey
+        </h2>
+        <div class="govuk-grid-row info-box__download-content">
+          <div class="govuk-grid-column-one-half">
+            <img src="/assets/images/preview-placeholder.jpg" alt="Placeholder image" class="dfe-file-preview-image">
+          </div>
+          <div class="govuk-grid-column-one-half">
+            <p class="govuk-body-s">
+              <strong>Resourced type:</strong> Template
+            </p>
+            <p class="govuk-body-s">
+              <strong>File type:</strong> Document, 70KB, 5 pages
+            </p>
+            <p>
+              <a class="govuk-link govuk-link--no-visited-state" href="#">
+                Download
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+{inset-text}
+
+“The online form made it accessible for staff. The responses were set out in a
+range of charts so I could explain clearly to staff what the outcomes were
+and indicate areas where we needed to focus”
+
+Claire Evans, Deputy Headteacher
+Anderton Park Primary School
+
+{/inset-text}
+
+## Suggestions for running a successful survey
+
+School leaders who hve conducted staff surveys in their own schools shared the
+following suggestions for running a successful survey:
+
+### Decide when and how often to run the survey
+
+Consider when in the school year it would be best to run your staff survey for
+it to have the most impact and allow you to act on the results. Carrying out
+regular surveys will allow you to evaluate the impact that your actions are
+having on staff workload, for example, some schools run these annually.
+
+### Decide who to survey and how long it will run for
+
+You could choose the survey a group of staff, for example middle leaders or
+teaching assistants and edit the survey to reflect their role.
+
+You should edit the survey to explain to staff how long the survey will remain
+open for.
+
+### Decide whether you need to anonymise the survey
+
+An anonymous survey may encourage more accurate responses from the staff who
+take part. If you want to understand differences in workload between teams or
+groups, questions about name and role can be added.
+
+### Decide what to do next
+
+You will need to consider the sequence of how you will act on the results of
+your staff survey. Give yourself time to analyse the results, update your staff
+and take action.
+
+You can use the [acting on your staff survey presentation](/workload-reduction-toolkit/identify-workload-issues/act-on-your-staff-survey)
```

Comment:
This [link](https://deploy-preview-57--iwwss-review.netlify.app/workload-reduction-toolkit/identify-workload-issues/act-on-your-staff-survey) didn't work:

<img width="494" alt="image" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/85567720/a68e02e5-6067-4c73-b02e-ec3f606e8e45">

Reply:
@mark-pkgflow resource not added yet! this is the first one in 'identify'